### ISSUE TITLE
Folder access automatically denied on signed app

### ISSUE THREAD

**Issue by zinne** [codebytere/node-mac-permissions#69]
Everything works fine. The permission popup appears in my dev env. But when I build the app with `electron-packager` and sign my app the permission popup does not appear. The log just returns a `denied` status.
(entitlements are setup correctly)
Can someone confirm that it works on the latest MacOS with a real signed app?

MacOS Sonoma 14.1
electron 26.4.2

**Comment by benrapport** [comment]
having this issue as well!

**Comment by sumersao** [comment]
hey @zinne I can confirm (for anything working in dev it works in production as well w/ a signed app). I was having this issue at first, but entitlements were my issue. Did you figure it out?

**Comment by zinne** [comment]
@sumersao no, didn‘t made it. Solved my situation just by using a save dialog.

**Comment by sumersao** [comment]
@zinne interesting. What permissions were you trying to use explicitly? fwiw the askForContactsAccess() didn't work for me in dev or prod, so I did a workaround by reading the resource directly and checking permissions that way. But other things worked

**Comment by zinne** [comment]
@sumersao I was trying to save a file to the Desktop.

**Comment by theDevelopper** [comment]
Have you defined the [NSDesktopFolderUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsdesktopfolderusagedescription) property in the info.plist?

**Comment by codebytere** [comment]
This is more than likely an entitlements issue - you should ensure you've defined all relevant properties as noted above.
